Gemfile.lock

```diff
@@ -96,6 +99,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```

Comment:
What are the diffs in this file for?

Reply:
It was made automatically by Jekyll with my version of ruby I can delete it i think.